frontend/src/app/features/work-packages/components/wp-fast-table/builders/rows/single-row-builder.ts

```diff
@@ -93,6 +97,10 @@ export class SingleRowBuilder {
       return this.relationCellBuilder.build(workPackage, column);
     }
 
+    if (column.id === 'sharedWithUsers') {
```

Comment:
To avoid referencing this magic value twice, you could add it to e.g., `frontend/src/app/features/work-packages/components/wp-fast-table/builders/internal-sort-columns.ts` and import it.

Reply:
done